### PR TITLE
Schema changes for Indirect relation between offer and User

### PR DESCRIPTION
**Pull request by ayushpandey8439**
Changes:

1. Add relation for Offer and Volunteer match request.
2. Add list type for offers in Volunteer model.

### CODE REVIEW

File: logs/jobs.2021-02-28.log [hunk 1]
```diff
@@ -0,0 +1,276 @@
+[2021-02-28T23:01:29.749] [INFO] default - Started Notification Handler
```
Comment (lsc0498) [inline logs/jobs.2021-02-28.log:1]: Can these log files be removed before merging? 🙂

**Review by lsc0498 — APPROVED** [review]
Looks good to me, thank you! 👍

File: logs/jobs.2021-02-28.log (hunk 1 above)
Comment (ayushpandey8439) [reply logs/jobs.2021-02-28.log:1]: I have removed them from the commits. I will get rid of these files too.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -61,6 +61,8 @@ model VolunteerMatchRequest {
     user        Volunteer   @relation(fields:[requestor],references:[id])
     requestor   String
     parameters  String[]
+    offer       Offer       @relation(fields:[offerId],references:[id])
+    offerId     String
```
[the remark this answers is not in the document]
Comment (ayushpandey8439) [reply prisma/schema.prisma:68]: The `offer` is just used to define the relation. The actual field in the database is the `offerId` which is mapped as a foreign key to the field `id` in the `Offer`

**Pull request merged**